src/Microsoft.AspNet.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -1135,17 +1140,19 @@ protected virtual string GenerateName(string expression)
         }
 
         protected virtual IHtmlContent GenerateValidationMessage(
+            ModelExplorer modelExplorer,
             string expression,
             string message,
-            object htmlAttributes,
-            string tag)
+            string tag,
+            object htmlAttributes)
```

Comment:
Switched the order of these parameters to better match `IHtmlGenerator.GenerateValidationMessage()`.